src/meta_memcache/interfaces/high_level_commands.py

```diff
@@ -19,6 +19,15 @@ def set(
     ) -> bool:
         ...  # pragma: no cover
 
+    def refill(
```

Comment:
It might be worth having documentation for this one since it's not really a built-in memcache command and more of a pattern the client introduces (I think?)